app/views/pseuds/_pseuds_form.html.erb

```diff
@@ -15,7 +15,7 @@
   
   <dt><%= f.label :description, ts("Description") %></dt>
   <dd>
-    <code>a href, em, strong, i, b</code>
+    <p class="html-notes"><%= allowed_html_instructions %></p>
```

Comment:
Can you remove `class="html-notes"`? It's just there for when we need to use JavaScript to toggle between HTML and RTE instructions on form fields that have an RTE option.